docs/source/examples/petstore/petstore.all.rego.compiled

```diff
@@ -58,6 +58,13 @@ line6_not1_cnd {
     input.potty_trained
 }
 
+deny {
+    seal_list_contains(seal_subject.groups, `everyone`)
+    input.verb == `buy`
+    re_match(`petstore.pet`, input.type)
+    input.tags["endangered"] == "true"
```

Comment:
Looking for petstore.all.rego and petstore.all.rego.compiled:
is expected rego for tags looks like `input.tags["name"]` or `input.tags.name` ?

Reply:
rego should look like:
```
deny {
...
    input.tags["endangered"] == "true"
}
```

Reply:
i.e. ctx.tags is a map of string to string

Reply:
@rchowinfoblox, @abynenkov-ib and i got on a zoom and determined we should generate tags with square brack quote notation and let opa fmt reformat it.

it would be confusing for a person writing a golang compiler to understand otherwise.